cluster/examples/kubernetes/rook-cluster.yaml

```diff
@@ -71,6 +83,15 @@ spec:
 #    - name: "172.17.4.101"
 #      directories:         # specific directores to use for storage can be specified for each node
 #      - path: "/rook/storage-dir"
+# you can override the cluster wide resource requests/limits per node too,
+# but only when not using `useAllDevices: true`!
```

Comment:
can you avoid the double negative? how about this?
- "but only when using `useAllDevices: false`"